### PR TITLE
Feature/unit testing phase1

### PR DESCRIPTION
**Pull request by fivetran-sheringuyen**
**What change does this PR introduce?** 
 
Adds a dbt compile test to our Fivetran Utils package

**If this PR introduces a new macro, how did you test the new macro?** 

**If this PR introduces a modification to an existing macro, which packages is the macro currently present in and what steps were taken to test compatibility across packages?** 

**Did you update the README to reflect the macro addition/modifications?** 

- [ ] Yes
- [x] No (provide further explanation)
Integration testing for internal uses

### CODE REVIEW

**Comment by fivetran-sheringuyen** [comment]
I will..squash before merge 😅

File: integration_tests/packages.yml
Comment (fivetran-joemarkiewicz) [inline integration_tests/packages.yml:11]: Should we maybe make these ranges larger? I would want to make sure we are testing on the latest versions of them.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-sheringuyen this is great and will really help us unsure that changes to this package have more coverage than just releasing and hoping a potential bug isn't introduced!

I have one small comment, but other than that - this looks great to go! I do want to hold off merging and releasing this until PR #100 is completed.

File: integration_tests/packages.yml
Comment (fivetran-sheringuyen) [reply integration_tests/packages.yml:11]: I think there's a couple of different things we could do here. 
1) Broaden the range as you are suggesting and having the upper bound be < 1.0.0
or
2) Having `revision: main` for the git reference. This ensures that it's always the latest version of main, although not necessarily a release. So there is a minor caveat here but for this context, I think it would be ok.

**Pull request merged**
